lib/linguist/languages.yml

```diff
@@ -1636,6 +1636,7 @@ Dockerfile:
   filenames:
   - Containerfile
   - Dockerfile
+  - Dockerfile.sim
```

Comment:
What?

Reply:
Somebody on GitHub has committed a file called Dockerfile.sim and it's Dockerfile syntax, not Simula syntax.

Reply:
lol

Reply:
You don't want to do this 😉 Every new addition needs supporting usage that meets our requirements and one single file you spotted and decided to add to this PR will block this PR. That user can use an override if they insist on using the `.sim` extension.

```suggestion
```